next/pages/channel.js

```diff
@@ -49,6 +49,12 @@ const AddChannelButton = styled(Button)`
   margin-left: auto;
 `
 
+const ChatBox = styled(Box)`
+  flex-basis: auto;
+  overflow-y: scroll;
+  height: 1px;
```

Comment:
@Selhar porque esse `1px` é necessário?

Reply:
Curiosamente eu também não sabia o porque e encontrei essa solução completamente por acidente, diminuindo o chat pra triggerar o scroll automaticamente. Perguntando no stack overflow, vi que é uma prática comum: https://stackoverflow.com/questions/54564976/grid-with-viewport-height-and-inner-scrolling-div

Basicamente o 1px é um parâmetro necessário pro flex-basis funcionar, flex-basis é relativo à algo, sem esse ponto de referência flex-basis não faz nada.